app/controllers/shelter_reviews_controller.rb

```diff
@@ -41,6 +41,11 @@ def update
     end
   end
 
+  def destroy
+    review = ShelterReview.destroy(params[:review_id])
+    redirect_to shelters_show_path(review.shelter.id)
+  end
+
```

Comment:
I like the format of this redirect_to path. Let's implement this throughout the app during a refactor if we have time.